systemtests/tests/python-bareos/test_list_command.py

```diff
@@ -258,15 +268,49 @@ def test_list_jobs(self):
         for job in result["jobs"]:
             self.assertEqual(job["type"], "B")
 
-        result = director.call("list jobs jobtype=Backup")
+        result = director.call("list jobs jobtype=Restore")
```

Comment:
Im not sure I understand this change.  Why did this happen ?

Reply:
to make sure jobtype `Restore` would actually return something. I tested the backup in the previous one

It tests two things at once: that the full status name gets correctly parsed and interpreted, and also that the restore is correctly listed.